src/components/CreateAnalysis/ConfirmAnalysis.tsx

```diff
@@ -14,7 +14,7 @@ const ConfirmAnalysis: React.FC<ConfirmAnalysisProps> = ({submit}) => {
 
   return (
     <div className="ConfirmAnalysis">
-      {dcmImages.map((img: DcmImage, i) => {
+      {dcmImages.filteredDcmImages.map((img: DcmImage, i) => {
```

Comment:
Should the ConfirmAnalysis component display all selected images regardless whether the filter has been applied or not?